TOOLS/osxbundle.py

```diff
@@ -45,8 +42,8 @@ def sign_bundle(binary_name):
         resolved_dir = os.path.join(bundle_path(binary_name), dir)
         for root, _dirs, files in os.walk(resolved_dir):
             for f in files:
-                sh('codesign --force -s - ' + os.path.join(root, f))
-    sh('codesign --force -s - ' + bundle_path(binary_name))
+                subprocess.call(['codesign', '--force', '-s', '-', os.path.join(root, f)])
```

Comment:
On newer (>3.5) python versions it is better to use `subprocess.run` instead. Also you could add `check=True` for run to check if there were no execution errors.

Reply:
thanks for the heads-up. i avoided using `check=True` in this case since there are/might be files in those folders that are not singable and would return a none 0 exit code (atm at least the .gitkeep file in the lib). i would look into this at later point of time and see if it can be done a bit more nicely.